gradio/components/login_button.py

```diff
@@ -39,14 +39,17 @@ def __init__(
         render: bool = True,
         scale: int | None = 0,
         min_width: int | None = None,
+        signed_in_value: str = "Signed in as {}",
     ):
         """
         Parameters:
-            signed_in_value: The text to display when the user is signed in. The string should contain a placeholder for the username, e.g. "Signed in as {}".
+            logout_value: The text to display when the user is signed in. The string should contain a placeholder for the username with a call-to-action to logout, e.g. "Logout ({})".
         """
-        if signed_in_value is None:
-            signed_in_value = "Signed in as {}"
-        self.signed_in_value = signed_in_value
+        if signed_in_value is not None and signed_in_value != "Signed in as {}":
```

Comment:
The typing says it can't be `None`. You might mean to
```suggestion
        if signed_in_value and signed_in_value != "Signed in as {}":
```

Reply:
Yes I saw that but wondered if some internal gradio mechanism was sometimes passing `None` as value so I kept it as it was done before (i.e. considering value could be none).

Otherwise I'm even fine to change to
```suggestion
        if signed_in_value != "Signed in as {}":
```

Reply:
@Wauplin This simpler version that assumes a string is fine. We probably just need to clean up any redundant conditional checks.

Reply:
Thanks for confirming!